cypress/e2e/inline-slots.cy.ts

```diff
@@ -1,25 +1,30 @@
 import { liveblogs, articles } from '../fixtures/pages';
+import { breakpoints } from '../fixtures/breakpoints';
 
 const pages = [...articles, ...liveblogs].filter(a => 'expectedMinInlineSlotsOnPage' in a);
 
 describe('Slots and iframes load on pages', () => {
 	pages.forEach(({ path, adTest, expectedMinInlineSlotsOnPage }) => {
-		it(`Test ${path} has at least ${expectedMinInlineSlotsOnPage} inline total slots`, () => {
-			cy.visit(`${path}?adtest=${adTest}`);
+		Object.entries(breakpoints).forEach(([breakpoint, width]) => {
+			it(`Test ${path} has at least ${expectedMinInlineSlotsOnPage} inline total slots at breakpoint ${breakpoint}`, () => {
+				cy.viewport(width, 500)
 
-			// Click "Yes, I'm happy" on the sourcepoint banner to obtain consent
-			cy.getIframeBody('sp_message_iframe_').find('.btn-primary').click();
-
-			cy.scrollTo('bottom', { duration: 5000 });
-
-			[...Array(expectedMinInlineSlotsOnPage).keys()].forEach(
-				(item, i) => {
-					cy.get(`[data-name="inline${i + 1}"]`).should(
-						'have.length',
-						1,
-					);
-				},
-			);
-		});
+				cy.visit(`${path}?adtest=${adTest}`);
+	
+				// Click "Yes, I'm happy" on the sourcepoint banner to obtain consent
+				cy.getIframeBody('sp_message_iframe_').find('.btn-primary').click();
+	
+				cy.scrollTo('bottom', { duration: 5000 });
+	
+				[...Array(expectedMinInlineSlotsOnPage).keys()].forEach(
```

Comment:
Is the spreading necessary?

Reply:
Good question. I think it is necessary because `Array.prototype.keys` returns an array iterator which we need to turn into an array by spreading before we can call `forEach` on it.
It might be slightly clearer here to do a for loop like `for (let i = 0; i < expectedMinInlineSlotsOnPage; i += 1) { /* ... */ }` but I think we tend to favor `forEach` as a rule.